AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -121,6 +121,7 @@ kraj.by#$#div[aria-labelledby="CModalLabel"] { display: none !important; }
 kraj.by#$#div[aria-labelledby="CModalLabel"] ~ div.modal-backdrop { display: none !important; }
 kraj.by#$#body { overflow: auto !important; padding-right: 0 !important; }
 sapatinhodeluxo.com.br##.bar-cookies
+nike.com#?##gen-nav-commerce-header-v2 > div.pre-modal-window:has(> .pre-modal > .pre-modal-view > .pre-cookie-modal-body)
```

Comment:
Broken page -> https://www.nike.com/pl/w/nowosci-mezczyzni-buty-3n82yznik1zy7ok

Reply:
Try `nike.com#%#//scriptlet('set-cookie', 'sq', '3')`

Reply:
broken was only scrolling?

Reply:
Eh, this pull request was merged, but the scriptlet rule is ok here.
Have you found any issues with css rules? I mean
```
nike.com#?##gen-nav-commerce-header-v2 > div.pre-modal-window:has(> .pre-modal > .pre-modal-view > .pre-cookie-modal-body)
nike.com#$#body[data-overlay="cookieModalSimple"] { position: static !important; }
```

Reply:
Now I can't found issues with frozen scripts on pages (e.g. breakage to add product into "shop bag") or glitched pop-up based on node/DOM `.pre-modal-window` (aka glitch [AdguardTeam/ExtendedCss/issues/122](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fgithub.com%2FAdguardTeam%2FExtendedCss%2Fissues%2F122))

I suppose newsletter may block scroll too in categories but now I can't load to found used `[data-overlay]`.

Reply:
Thank you, I'll change it to your rules in the future, if I find something wrong with the current rule.

Reply:
ExntendedCSS may no longer safe:  #88423